### ISSUE TITLE
Classic NES Series games flickering

### ISSUE THREAD

**Issue by Cibomatto2002** [visualboyadvance-m/visualboyadvance-m#61]
### Expected behavior

No flickering 

### Actual behavior

Flickering 

### Steps to reproduce the behavior

Play Classic NES Series - Excitebike (USA, Europe), Classic NES Series - Metroid (USA, Europe), Classic NES Series - Super Mario Bros. (USA, Europe).

### Build details

VisualBoyAdvance-M-WX-2.0.0-beta2

Operating System (Windows, Mac, Linux, etc.):
Windows 10 64-Bit

Interface (wx, SDL, default is wx):
Default

Version of code (pre-built binary version, or commit ref, or just "master"):
I think master

Any other relevant build information:
No

If this is e.g. a joystick or video issue, then the relevant hardware information:

Intel HD Graphics (Gigabyte)
1023MB NVIDIA GeForce GT 720 (EVGA)
ForceWare version: 365.10
SLI Disabled

**Comment by DoctorWho11** [comment]
Try this beta 3 release https://github.com/visualboyadvance-m/visualboyadvance-m/releases/download/Beta-3/visualboyadvance-m-win32-Beta-3.7z

**Comment by Cibomatto2002** [comment]
It's better but Classic NES Series - Excitebike (USA, Europe)  still has it.  Classic NES Series - Super Mario Bros. (USA, Europe) has a little.

**Comment by rkitover** [comment]
@Cibomatto2002 Have you tried both the Simple and OpenGL graphics output drivers?

**Comment by Cibomatto2002** [comment]
Yes both Simple and OpenGL do it. I made a video to explain it better to you all.

https://www.youtube.com/watch?v=-lDL5hCYTCs&feature=youtu.be

**Comment by Squall-Leonhart** [comment]
reproduce on a current nvidia display driver please.

**Comment by retro-wertz** [comment]
im not sure how these classic roms should look like in gba. some dont flicker others do like Super Mario. other emu flickers as well even the advertised accurate ones.

**Comment by negativeExponent** [comment]
this is still an issue with latest commit https://github.com/visualboyadvance-m/visualboyadvance-m/commit/b23d463652da94579381be5a00c12909704cfb04

strangely enough, running the latest libretro core using this same commit does not have the said flickering issue (more obvious in Classic SMB)

interestingly, running vbam with either interframe-blend modes other than none remedies the problem, both in simple or gl, while this is not required on the libretro port (even if core is run in 32bpp mode)

**Comment by Squall-Leonhart** [comment]
these games are supposed to be interframe blended.

**Comment by negativeExponent** [comment]
as i have stated, libretro, which uses the same commit does not need
interframe blending (or lcd ghosting effect) to be enabled. Its rare to
have a gba game uses such slow refresh to create some effects (only see a
few games that may need it like race timers etc). But it should never be
flicking like it does on the standalone compared to libretro or other emu.
this is not the same flickering on gb/gbc that you forced to dismiss before
(which the issue was not even about the flickering).

**Comment by Squall-Leonhart** [comment]
i don't care what libretro does.
